src/Spice86.Core/Emulator/OperatingSystem/Structures/DosDiskTransferArea.cs

```diff
@@ -7,6 +7,8 @@
 /// Represents a DTA (Disk Transfer Area) in memory.
 /// </summary>
 public class DosDiskTransferArea : MemoryBasedDataStructure {
+    private const int ResultIdOffset = 0x02;
```

Comment:
Move it with the other offsets? Also, I can't find where is the corresponding field used.

Reply:
Ah yes, this is used by my C# implementation of the [StdIO C library](https://github.com/JorisVanEijden/ReverseBak/blob/f3c6fb5bb86c8bff84ccfabc1a136f79e331965f/BetrayalAtKrondor/Overrides/Libraries/StdIO.cs#L292) used by Krondor. 

Max introduced the "SearchId" later to fix it in spice. But that is marked `internal` so my library can't use it.
I'll just copy this version to my library and leave it alone in Spice86.

Reply:
If you have a use for this field, why not putting it here and making the lib fix max did use it?
Internal could be removed, in assembly nothing is internal :smile:

Reply:
Nah, it's fine.
All the file handling in Krondor is done through StdIO. So I csharpified that to completely bypass the whole DOS (INT 21) and go straight to the host filesystem.
I just "borrowed" that DTA structure, but it makes more sense to also move that to the StdIO lib, since that was the last tie to DOS :)